updaterecord.php

```diff
@@ -0,0 +1,4 @@
+<?php
+require_once 'vendor/autoload.php';
```

Comment:
If this is all that this file does, you should instead remove the file, and put the header function into index instead of `header('Location: updaterecord.php');`